CHANGELOG.md

```diff
@@ -53,6 +53,7 @@
 - Added new `--version` flag to the `operator-sdk scorecard` command to support a new output format for the scorecard. ([#1916](https://github.com/operator-framework/operator-sdk/pull/1916)
 - Added new `--selector` flag to the `operator-sdk scorecard` command to support filtering scorecard tests based on labels added to each test. ([#1916](https://github.com/operator-framework/operator-sdk/pull/1916)
 - Added new `--list` flag to the `operator-sdk scorecard` command to support listing scorecard tests that would be executed based on selector filters. ([#1916](https://github.com/operator-framework/operator-sdk/pull/1916)
+- For scorecard version v1alpha2 only, return code logic was added to return 1 if any of the selected scorecard tests fail.  A return code of 0 is returned if all selected tests pass. ([#1916](https://github.com/operator-framework/operator-sdk/pull/1916)
```

Comment:
```suggestion
- Change points result for pass/fail in the scorecard version v1alpha2 only. ([#1916](https://github.com/operator-framework/operator-sdk/pull/1916)
```
WDYT? something more user-friendly ;-).

Reply:
I like the current comment better I think, this change is about the exit code that scorecard returns instead of a change to the points.